File: home/src/main/java/com/example/home/HomeActivity.java
package com.example.home;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.alibaba.android.arouter.launcher.ARouter;
import com.example.base.BaseActivity;
import com.example.base.RouteUtils;


@Route(path = RouteUtils.HOME)
public class HomeActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.home_main);
        findViewById(R.id.tv).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ARouter.getInstance().build(RouteUtils.HOME1).navigation();
            }
        });
    }
}

File: home/src/main/java/com/example/home/HomeActivity.kt
package com.example.home

import android.os.Bundle
import android.view.View
import com.alibaba.android.arouter.facade.annotation.Route
import com.alibaba.android.arouter.launcher.ARouter
import com.example.base.BaseActivity
import com.example.base.RouteUtils

@Route(path = RouteUtils.HOME)
class HomeActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.home_main)
        findViewById<View>(R.id.tv).setOnClickListener {
            ARouter.getInstance().build(RouteUtils.HOME1).navigation()
        }
    }
}